data/remnant/remnant missions.txt

```diff
@@ -2166,3 +2166,298 @@ mission "Remnant: Face to Maw 2B"
+
+
+
+mission "Remnant: Crystal Research 5"
+	name "Scan the <npc> in <waypoints>"
+	description `Go scan the <npc> to record the Keystone post-wormhole.`
+	landing
+	waypoint "Peragenor"
+	source "Viminal"
+	destination "Viminal"
+	npc "scan outfits" "save"
+		government "Remnant"
+		personality coward disables plunders staying surveillance target mining
+		system "Peragenor"
+		ship "Starling" "Winds of Light"
+	to offer
+		has "Remnant: Crystal Research 4: done"
+	on offer
+		conversation
+			`	"Ready? Let's go!" signs the starling pilot as he fires up his systems and gestures an acknowledgement to the dockworker. The worker nimbly steps clear of the ship and watches as the two ships head back out of the dome.`
+				launch
+	on visit
+		dialog `You land on <planet>, but you haven't scanned the <npc> in <waypoints>. Make sure you scan the ship before returning.`
+	on complete
+		payment 150000
+		conversation
+			`The scanning proceeds smoothly, and while it does note some differences between the two crystals, they both appear to function similarly. After you return to Viminal the pilot comes over to collect a copy of the data for the researchers.`
+			`	"Thanks for your help, Captain <first>. This information may be valuable in understanding how the wormholes work." He pauses, then continues thoughtfully "Perhaps more importantly, it may help understand how the wormholes of the Ember Waste are different from the wormholes elsewhere that require no such aids."`
+			`	He looks down at the data crystal. "Before I forget, the outfitter said he would like to meet you. He should be in the cafeteria waiting for you. May the Embers burn bright for you, Captain." He gives a wave that seems vaguely like a salute and heads off.`
+			`	As he disappears from sight you hear a faint chime from your commlink, which displays a message indicating that two payments of <payment> have been allocated to your account. You recall that you didn't get a message after the last mission, so they must have been combining them. You do find it odd, however, that they always seem to know exactly when you finish a delivery.`
+
+
+
+mission "Remnant: Crystal Research 6"
+	name "Deliver Data to <destination>"
+	description `A copy of the scanner data needs to be delivered to the researchers on <destination>.`
+	source "Viminal"
+	destination
+		government "Remnant"
+	to offer
+		has "Remnant: Crystal Research 5: done"
+	on offer
+		conversation
+			`You walk into the cafeteria and spot the outfitter halfway through an impressively large meal. He waves you over, and by the time you are seated a tray with a significantly smaller meal is waiting for you. While you have aclimitized to a few of the dishes, most of them are still very unappetizing.`
+			choice
+				`	"Remnant sure like to eat."`
```

Comment:
this and the choice below it, as I understood it you were trying to go for something like "Why do Americans [do X]?", in referring to the Remnant as a sort of regional people, yes? I don't think it quite works like that though, dunno how to explain it but the word Remnant on both choices feel off, like something is missing to make them work and all. Guess an example would be if someone just said "British like to drink tea", which would seem odd compared to "The british like to drink tea", but I don't know if that'd help with the Remnant case. maybe "you Remnant" ? but then that might seem kinda douchey from <first> <last> I guess so idk.

Reply:
That's a good point. I don't think adding "The" in makes it any worse, and it does potentially make it better, so changed it.